biogeophys/EDSurfaceAlbedoMod.F90

```diff
@@ -195,756 +125,918 @@ subroutine ED_Norman_Radiation (nsites, sites, bc_in, bc_out )
                     bc_out(s)%fabd_parb(ifp,:) = 0.0_r8
                     bc_out(s)%fabi_parb(ifp,:) = 0.0_r8
                     do ib = 1,hlm_numSWb
+
+                       ! Requires a fix here, abld vs albi
```

Comment:
so are we going to fix this here, or in https://github.com/NGEET/fates/pull/431/files#diff-68b652d39e00b970fbcf3d5b8d6f1165L199 ?

Reply:
i was going to fix this, but didn't realize @jkshuman got to it first until I saw that PR.  I think it depends on which one gets integrated first.  There will be a conflict between the two either way (so they both will?)
For testing purposes, I will first run tests maintaining the status quo to test b4b.  Then I will implement that fix.